lib/titiler-pgstac-api/runtime/src/handler.py

```diff
@@ -8,9 +8,7 @@
 from mangum import Mangum
 from utils import get_secret_dict
 
-pgstac_secret_arn = os.environ["PGSTAC_SECRET_ARN"]
-
-secret = get_secret_dict(pgstac_secret_arn)
+secret = get_secret_dict(secret_arn_env_var="PGSTAC_SECRET_ARN")
```

Comment:
Same here we can define the settings https://github.com/stac-utils/titiler-pgstac/blob/b2aa992554f93acda41473a5eb5e9688ee0a460b/titiler/pgstac/db.py#L13

and pass it to `connect_to_db` function

Reply:
Yes we could do that but it is impossible to import anything from `titiler.pgstac.main` without the postgres environment variables set because [`assemble_db_connection`](https://github.com/stac-utils/titiler-pgstac/blob/main/titiler/pgstac/settings.py#L77-L90) gets run on every new instance of `PostgresSettings` and it will fail. Another option here would be to set placeholder values in those environment variables in the Lambda configuration then update it in the runtime but that could be a trap for someone who uses a custom runtime.

Reply:
I'm going to push a fix and release a new version without the initialization of the settings